src/devices/custom_devices_diy.ts

```diff
@@ -1405,6 +1405,129 @@ const definitions: Definition[] = [
             }
         },
     },
+    {
+        // https://github.com/devbis/z03mmc/
+        zigbeeModel: ['LYWSD03MMC-z'],
+        model: 'LYWSD03MMC-z',
+        vendor: 'Custom devices (DiY)',
+        description: 'Xiaomi temperature & humidity sensor with custom firmware',
+        extend: [
+            quirkAddEndpointCluster({
+                endpointID: 1,
+                outputClusters: [],
+                inputClusters: [
+                    'genPowerCfg',
+                    'msTemperatureMeasurement',
+                    'msRelativeHumidity',
+                    'hvacUserInterfaceCfg',
+                ],
+            }),
+            batteryPercentage(),
+            temperature({reporting: {min: 10, max: 300, change: 10}}),
+            humidity({reporting: {min: 10, max: 300, change: 50}}),
+            enumLookup({
+                name: 'temperature_display_mode',
+                lookup: {'celsius': 0, 'fahrenheit': 1},
+                cluster: 'hvacUserInterfaceCfg',
+                attribute: 'tempDisplayMode',
+                description: 'The units of the temperature displayed on the device screen.',
+            }),
+            binary({
+                name: 'show_smiley',
+                valueOn: ['SHOW', 1],
+                valueOff: ['HIDE', 0],
+                cluster: 'hvacUserInterfaceCfg',
+                attribute: {ID: 0x0010, type: dataType.boolean},
+                description: 'Whether to show a smiley on the device screen.',
+            }),
+            binary({
+                name: 'enable_display',
+                valueOn: ['ON', 1],
+                valueOff: ['OFF', 0],
+                cluster: 'hvacUserInterfaceCfg',
+                attribute: {ID: 0x0011, type: dataType.boolean},
+                description: 'Whether to turn display on/off.',
+            }),
+            numeric({
```

Comment:
Calibration settings are wrong for this fw

Reply:
It seems this custom quirk is the current 'official' quirk: https://github.com/pvvx/ZigbeeTLc/blob/master/z2m/lywsd03mmc-z.js